inject/src/LoadLibraryR.c

```diff
@@ -206,15 +206,19 @@ HANDLE WINAPI LoadRemoteLibraryR( HANDLE hProcess, LPVOID lpBuffer, DWORD dwLeng
 			if( !dwReflectiveLoaderOffset )
 				break;
 
-			// alloc memory (RWX) in the host process for the image...
-			lpRemoteLibraryBuffer = VirtualAllocEx( hProcess, NULL, dwLength, MEM_RESERVE|MEM_COMMIT, PAGE_EXECUTE_READWRITE ); 
+			// alloc memory (RW) in the host process for the image...
+			VirtualProtectEx = VirtualAllocEx( hProcess, NULL, dwLength, MEM_RESERVE|MEM_COMMIT, PAGE_READWRITE ); 

```

Comment:
This should be setting lpRemoteLibraryBuffer with the return value from VirtualAllocEx?